app/views/shared/samples/spreadsheet_imports/_dialog.html.erb

```diff
@@ -0,0 +1,113 @@
+      <% end %>
+    >
+
+      <%= form_for(:spreadsheet_import, url: url, method: :post) do |form| %>
+        <input type="hidden" name="broadcast_target" value="<%= @broadcast_target %>"/>
+        <div class="grid gap-4">
+          <p class="text-base leading-relaxed text-slate-500 dark:text-slate-400">
+            <%= t(".description") %>
+          </p>
+          <p class="text-base leading-relaxed text-slate-500 dark:text-slate-400">
+            <% if @namespace.type == 'Group' %>
+              <%= t(".namespace.group.description_html") %>
+            <% else %>
+              <%= t(".namespace.project.description_html") %>
+            <% end %>
+          </p>
+          <div class="form-field">
+            <%= form.label :file,
+                       t(".file"),
+                       class:
+                         "block mb-2 text-sm font-medium text-slate-900 dark:text-white" %>
+            <%= form.file_field :file,
+                            required: true,
+                            accept:
+                              # https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+                              "text/csv,.tsv,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+                            data: {
+                              action: "change->spreadsheet-import#readFile",
+                            },
+                            class:
+                              "block w-full text-sm text-slate-900 border border-slate-300 rounded-lg cursor-pointer bg-slate-50 dark:text-slate-400 focus:outline-none dark:bg-slate-700 dark:border-slate-600 dark:placeholder-slate-400" %>
+            <p class="mt-1 text-sm text-slate-500 dark:text-slate-300"><%= t(".file_help") %></p>
+          </div>
+          <div class="form-field">
+            <%= form.label :sample_name_column,
+                       t(".sample_name_column"),
+                       class:
+                         "block mb-2 text-sm font-medium text-slate-900 dark:text-white
+                         disabled:bg-slate-50 disabled:border disabled:border-slate-300 disabled:rounded-lg disabled:focus:ring-blue-500 disabled:focus:border-blue-500 disabled:w-full disabled:p-2.5 disabled:dark:bg-slate-700 disabled:dark:border-slate-600 disabled:dark:placeholder-slate-400 disabled:dark:focus:ring-blue-500 disabled:dark:focus:border-blue-500" %>
+            <%= form.select :sample_name_column,
+                        {},
+                        { prompt: t(".select_sample_name_column") },
+                        required: true,
+                        data: {
+                          "spreadsheet-import-target": "sampleNameColumn",
+                          action:
+                            "change->spreadsheet-import#changeSampleNameInput",
+                        } %>
+          </div>
+          <% if @namespace.type == 'Group' %>
```

Comment:
Same comment as above, can change to `@namespace.group_namespace?` and maybe move the code from within this if block into another partial

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/984/commits/1c30d7b0e5a3d125104197aab4e8b2066f557f74